website/docs/sql_ddl.md

```diff
@@ -383,18 +383,57 @@ CREATE CATALOG hoodie_catalog
 The following is an example of creating a Flink table. Read the [Flink Quick Start](/docs/flink-quick-start-guide) guide for more examples.
```

Comment:
Can we follow a similar pattern as the Spark SQL one with a syntax first and then followed by the examples? I believe there can be one syntax for the create table section followed by partitioned and non-partitioned sections.